custom/icds_reports/utils/__init__.py

```diff
@@ -199,7 +221,9 @@ def _custom_data(self, selected_location, domain):
 
             timer = TimingContext()
             with timer:
-                report_data = ICDSData(domain, filters, config['id']).data()
+                allow_conditional_agg = self.allow_conditional_agg and not config.get('disallow_conditional_agg', False)
```

Comment:
double negatives like `not disallow_...` can be tricky to follow. Could this be changed to something like `... and config.get('allow_conditional_agg', True)`?

Reply:
Fair point and considered it myself, but there are only a couple which require `disallow_conditional_agg` (4~5) flag vs a many more (~30) that would require `allow_conditional_agg` flag.